crates/bdk/tests/wallet.rs

```diff
@@ -1663,9 +1686,10 @@ fn test_bump_fee_reduce_single_recipient() {
         .insert_tx(tx, ConfirmationTime::Unconfirmed { last_seen: 0 })
         .unwrap();
 
+    let feerate = FeeRate::from_sat_per_kwu(625); // 2.5 sat/vb
```

Comment:
The math is not matching for me.
Am I doing something wrong?

`dbg!(psbt.clone().extract_tx().weight().to_wu() as f64 * 2.5);`

returns `830.0`